src/pages/settings/components/dashboard/background/modal.tsx

```diff
@@ -0,0 +1,61 @@
+import { XMarkIcon } from '@heroicons/react/24/outline'
+import React, { FC, useState } from 'react'
+
+import { Button } from 'components/button'
+import { FileInput } from 'components/fileInput'
+
+import { Colors } from 'const/enums'
+import { useAppDispatch } from 'store'
+import { updateUserProfileThunk } from 'store/thunks/user/user.thunk'
+
+import { Portal } from 'utils/portal'
+
+import { DashboardModalImage, DashboardModalTitle, DashboardModalWrapper } from './styles'
+
+interface DashboardModalProps {
+  handleCloseModal: VoidFunction
+  userId: string
+}
+
+export const DashboardModal: FC<DashboardModalProps> = ({ handleCloseModal, userId }) => {
```

Comment:
Naming should indicate only what contains this component, not where it placed. Please rename it